setup.py

```diff
@@ -1,10 +1,76 @@
+import os
```

Comment:
I would suggest to take the opportunity to add headers for copyright and license to all files.